src/BuiltInTools/dotnet-watch/HotReloadDotNetWatcher.cs

```diff
@@ -294,6 +299,13 @@ private static void ConfigureExecutable(DotNetWatchContext context, ProcessSpec
             }
         }
 
+        private string GetRelativeFilePath(string path)
+        {
+            return path.StartsWith(_workingDirectory) ?
```

Comment:
```suggestion
            return path.StartsWith(_workingDirectory, StringComparison.OrdinalIgnoreCase) ?
```

Reply:
I had considered this (because MacOS is case insensitive), however Linux is case sensitive and Windows is configurable. Going to use `StringComparison.Ordinal` to be safe.